### DISCUSSION TITLE
Sort by following

### DISCUSSION THREAD

**Discussion by starmanv4** [community/community#140151] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Here are some sort option when we view user list.
But that doesn't include a option by number of following.
How can I implement this?

**Comment by masterwww1** [comment]
Here is my answer.

SELECT * 
FROM users 
ORDER BY following_count DESC;

**Reply by starmanv4** [reply]
No!!!
This is not the answer about my question.
I mean, how can I search users by most following in GitHub?

**Reply by masterwww1** [reply, marked as answer]
Unfortunately, GitHub’s built-in search functionality doesn’t allow you to sort by the number of followers.
But here is another way to search.
It is not on GitHub.
Only require GitHub API.


Here is code;:::



import requests

Replace 'username' with a GitHub username
response = requests.get('https://api.github.com/users/{username}')
data = response.json()
print(f"{data['login']} has {data['followers']} followers.")

**Reply by Ikon249** [reply]
Gyib

**Reply by Ikon249** [reply]
Rxdd

**Reply by Ikon249** [reply]
Dyeuj
